packages/contracts-bedrock/src/L2/RevenueSharer.sol

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
+
+import { L2StandardBridge } from "src/L2/L2StandardBridge.sol";
+import { FeeVault } from "src/universal/FeeVault.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { SafeCall } from "src/libraries/SafeCall.sol";
+
+/// @custom:proxied
+/// @custom:predeploy 0x4200000000000000000000000000000000000024
+/// @title RevenueSharer
+/// @dev Withdraws funds from system FeeVault contracts,
+/// pays a share of revenue to a designated Beneficiary
+/// and sends the remainder to a configurable adddress on L1.
+contract RevenueSharer {
```

Comment:
Just want to clarify that the L2 proxy admin owner can cause the revshare predeploy to halt by changing the fee vault config